src/Test.FunctionalTests.BinSkim.Driver/BaselineTestData/Expected/MixedMode_x64_VS2013_Default.dll.sarif

```diff
@@ -882,14 +882,14 @@
               "id": "BA2024",
               "name": "EnableSpectreMitigations",
               "fullDescription": {
-                "text": "Application code should be compiled with the Spectre mitigations switch (/Qspectre) and toolsets that support it."
+                "text": "Application code should be compiled with the Spectre mitigations switch (/Qspectre cl.exe command-line argument or <SpectreMitigation>Spectre</SpectreMitigation> build property). Spectre attacks can compromise hardware-based isolation, allowing non-privileged users to retrieve potentially sensitive data from the CPU cache. To resolve this issue, provide the /Qspectre switch on the compiler command-line (or specify <SpectreMitigation>Spectre</SpectreMitigation> in build properties), or pass /d2guardspecload in cases where your compiler supports this switch and it is not possible to update to a toolset that supports /Qspectre. This warning should be addressed for code that operates on data that crosses a trust boundary and that can affect execution, such as parsing untrusted file inputs or processing query strings of a web request. You may need to install the 'C++ spectre-mitigated libs' component from the Visual Studio installer if you observe violations against C runtime libraries such as libcmt.lib, libvcruntime.lib, etc."
```

Comment:
All the file changes under **BaselineTestData/Expected** are generated by the tool: **UpdateBaselines.cmd.**
The change should go with commit (https://github.com/microsoft/binskim/commit/5f9aca8676f6c39aaf230f637b57fa414c0c3f27) . It won't cause test failure under netcore3.1 but will cause test failure in net6.0.